packages/offix-datastore/package.json

```diff
@@ -16,6 +16,7 @@
     "build": "tsc",
     "lint-fix": "eslint \"./src/**/*.ts\" \"./tests/**/*.ts\" --fix",
     "test": "jest",
+    "watch": "tsc --watch",
```

Comment:
you can now watch changes so app will reload when you make any change in datastore